lib/webdriveragent.js

```diff
@@ -149,6 +150,32 @@ class WebDriverAgent {
 
     await B.all(stops);
   }
+
+  //Body must be object
+  async sendCommandWithSession(command, body, method) {
```

Comment:
Can you help me understand how this is already handled? Basically I wanted a way to take the incoming request, not proxy it automatically, translate it to how I wanted it and send it along to WebDriverAgent. i'd be happy not to have to make this new method, I just didn't see a way to accomplish the interception then modification then resending, with existing code.